export.md

```diff
@@ -21,7 +21,11 @@ Note: Only data the client application has authorization to access and that the
 
 ```GET [fhir base]/Group/[id]/$export```
 
-FHIR Operation to obtain data on all patients listed in a single [FHIR Group Resource](https://www.hl7.org/fhir/group.html). Note: How these groups are defined will be implementation specific for each clinical system. For example, a payer may send a healthcare institution a roster file that can be imported into their EHR to create or update a FHIR group. Group membership could be based upon explicit attributes of the patient, such as: age, sex or a particular condition such as PTSD or Chronic Opioid use, or on more complex attributes, such as a recent inpatient discharge or membership in the population used to calculate a quality measure. Although, FHIR based group management is out of scope for the bulk data project, it would be a valuable project.
+FHIR Operation to obtain data on all patients listed in a single [FHIR Group Resource](https://www.hl7.org/fhir/group.html). 
+
+If a server support getting data for Group it SHOULD support reading and shearching the Group resource summaries. This would allow clients to discover available groups and not store directly group IDs. 
```

Comment:
nit: typo 

> SHOULD support reading and s~~h~~earching the Group

Reply:
Thanks, fixed

Reply:
Last bullet should read "If a server supports getting data for a Group, it SHOULD support reading..."
(support => supports and added a comma for the if,then)